### DISCUSSION TITLE
Inquiry About Current TODOs and Future Plans for the Project

### DISCUSSION THREAD

**Discussion by xwEric** [cashapp/redwood#2228] [Q&A]
Hello,

I hope this message finds you well! We are considering integrating your framework into our production environment, including on iOS,Android,Web platforms. To better understand how we might proceed, could you kindly share the current TODOs and future milestones for the project?

Additionally, if there are any significant unresolved issues or challenges anticipated in the near future, having that information would be incredibly helpful. Your insights will greatly assist us in planning our integration strategy effectively.

Looking forward to your response and thank you in advance for your help!

**Comment by JakeWharton** [comment]
There is no high-level roadmap or milestones planned at this time. Releases are done monthly just to get something out there.

We use Redwood and Treehouse in production on Android and iOS at Cash, and you can hear about some of the problems in this talk that Jesse and I gave a few years ago: https://jakewharton.com/native-ui-with-multiplatform-compose/.

The project is open source because it keeps us honest, but it's not really ready for public consumption. There's basically no documentation, and entire frontends are missing real support (Compose UI, Swift UI). If you adopt it today you are signing up for a lot of work.

**Reply by JackieBuxiaohui** [reply]
hi The video in [ https://jakewharton.com/native-ui-with-multiplatform-compose/ ]cannot be played. Is there any other link？

**Reply by JakeWharton** [reply]
You can try https://www.droidcon.com/2022/09/29/native-ui-with-multiplatform-compose/, but it's the same video embed as the one on my site.
